pkgs/os-specific/linux/kernel/common-config.nix

```diff
@@ -251,7 +251,8 @@ let
       # (stable) amdgpu support for bonaire and newer chipsets
       DRM_AMDGPU_CIK = whenAtLeast "4.9" yes;
       # amdgpu support for RX6000 series
-      DRM_AMD_DC_DCN3_0 = whenAtLeast "5.9.12" yes;
+      DRM_AMD_DC_DCN3_0 = whenBetween "5.9.12" "5.11" yes;
+      DRM_AMD_DC_DCN = whenAtLeast "5.11" yes;
```

Comment:
Based on the commit message in https://github.com/torvalds/linux/commit/20f2ffe504728612d7b0c34e4f8280e34251e704, it looks like `DRM_AMD_DC_DCN3_0` was folded into `DRM_AMD_DC_DCN`.

Should we still have this enabled for 5.11?